src/common/timer/timer.js

```diff
@@ -0,0 +1,41 @@
+export class Timer {
```

Comment:
Having directories that only contain a single file is a little annoying. Just a personal opinion.